app/models/user.rb

```diff
@@ -1,2 +1,7 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id', class_name: 'Post'
```

Comment:
- [ ] Great job so far @over-geek There are a few changes I will suggest you implement and you'll be on your way to the next Milestone :+1: 
- [ ] Per the project requirements, you are to ensure that the associations between models are set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) and from the ERD diagram, you can see that the `User Table` has a one to many relationships with `Posts`, `Comments` and `Likes` but currently in the `User Table`, only the association between users and posts have been set. So kindly fix that :wink:

Reply:
- Well resolved